stacker/stack.py

```diff
@@ -57,7 +57,7 @@ def __repr__(self):
 
     @property
     def requires(self):
-        requires = set(self.definition.get('requires', []))
+        requires = set(map(self.context.get_fqn, self.definition.get('requires', [])))
```

Comment:
So, a style thing, but I think a list comprehension would be easier to read here:

```
requires = [self.context.get_fqn(r) for r in self.definition.get('requires', [])]
```